cmd/nodepool/update.go

```diff
@@ -18,14 +18,15 @@ var (
 	}
 
 	updateOpts = struct {
-		awsDebug bool
-		s3URI    string
+		awsDebug, prettyPrint bool
+		s3URI                 string
 	}{}
 )
 
 func init() {
 	NodePoolCmd.AddCommand(cmdUpdate)
 	cmdUpdate.Flags().BoolVar(&updateOpts.awsDebug, "aws-debug", false, "Log debug information from aws-sdk-go library")
+	cmdUpdate.Flags().BoolVar(&updateOpts.prettyPrint, "prettyPrint", false, "Pretty print the resulting CloudFormation")
```

Comment:
@c-knowles I like this more if it were `pretty-print` instead of `prettyPrint` for consistency with other flags like `aws-debug` 😊